### ISSUE TITLE
linter: LSP warnings without Oxlint config

### ISSUE THREAD

**Issue by cpojer** [oxc-project/oxc#16254]
### What version of Oxlint are you using?

latest

### What command did you run?

VS Code lsp

### What does your `.oxlintrc.json` config file look like?

None

### What happened?

When using Oxfmt but not Oxlint with the VS Code "Oxc" extension, it triggers warnings for various default rules:

<img width="1220" height="283" alt="Image" src="https://github.com/user-attachments/assets/77f885fe-cfb7-4085-a0e2-e84edbf12de1" />

I created an Oxlint config to ignore all files:

```
{
  "ignorePatterns": ["**/*.ts", "**/*.tsx"]
}
```

**Expected Behavior:** When using the Oxc extension only for formatting with Oxfmt and a project does not have an Oxlint configuration, it should not show any linter warnings/errors.

I recommend a separate conversation about the default rules and whether they should be warnings or errors.

**Comment by connorshea** [comment]
Misclicked and assigned myself; this should be resolved by https://github.com/oxc-project/oxc/pull/15837

**Comment by Sysix** [comment]
With the latest version(s) of VS Code extension, you can use only `oxfmt` without `oxlint`. Be sure there is no `node_modules/.bin/oxlint` file in your project, or you use `oxc.requireConfig` setting :)
